bootcfg/storage/filestore.go

```diff
@@ -112,7 +112,12 @@ func (s *fileStore) ProfileList() ([]*storagepb.Profile, error) {
 	return profiles, nil
 }
 
-// IgnitionGet gets an Ignition Config template by name.
+// IgnitionPut creates or updates an Ignition template.
+func (s *fileStore) IgnitionPut(name string, config []byte) error {
+	return Dir(s.root).writeFile(filepath.Join("ignition", name), config)
```

Comment:
You might want to escape these names before writing them to the filesystem (or validate them elsewhere) so you don't get names like "../../../vmlinuz"